doc/arkode/guide/source/Constants.rst

```diff
@@ -317,6 +317,20 @@ contains the ARKODE output constants.
    +-----------------------------------------------+------------------------------------------------------------+
    |                                               |                                                            |
    +-----------------------------------------------+------------------------------------------------------------+
+   | **LSRK method types**                         |                                                            |
+   +-----------------------------------------------+------------------------------------------------------------+
+   | :index:`ARKODE_LSRK_RKC`                      | Use Runge-Kutta-Chebyshev (RKC) method.                    |
```

Comment:
The RKC and RKL constants should have the order specified in the constant name and in the description to allow for other orders to be added later.